.github/workflows/labeler.yml

```diff
@@ -0,0 +1,18 @@
+# Workflow to associate labels automatically
+name: labeler
+# Trigger the workflow on pull request events
+on: [pull_request]
+jobs:
+  label:
+    runs-on: ubuntu-18.04
+    steps:
+      # We need to checkout the repository to access the configured file (.github/label-pr.yml)
+      - uses: actions/checkout@v1
```

Comment:
Why we are not using the V2 here? We need to take care of the `fetch-depth`. It seems implemented right now, but one of the first releases I tested on the V2 did not have this option (that was limited to the V1)?

Reply:
I changed to v2 indeed ;-) Good catch. I remove the `fetch-depth` as the default value (`1`) is enough. We don't expect much history as we just need to get the `label-pr.yml` file within the last commit.